src/cryptoadvance/specter/hwi_rpc.py

```diff
@@ -0,0 +1,258 @@
+from flask import Flask, url_for, render_template, jsonify, request, make_response, redirect, Response, current_app
```

Comment:
why do we need this imports here?
I would like to keep `hwi_rpc.py` flask-independent

Reply:
I removed it from there. It is still dependent on having access to the Specter instance to get the network and extract keys (add them to the Device Manager). Do you think it's better to separate it too so that the HWI part is completely independent from Specter itself?

Reply:
I think it might also make sense to refactor the extract xpubs so that instead of having a separate screen for that, it'll just be a popup like the rest of HWI functionalities.
So the `Get over USB`  button we currently have will just start the process instead of going to a separate screen. What do you think?

Reply:
Making xpub extraction with a popup is a good idea!
We want to extract xpubs both for testnet and mainnet, so I think it would be better to pass testnet flag or chain parameter with the request.